src/NodeJS/Server.php

```diff
@@ -338,31 +339,40 @@ public function start(Process $process = null)
     {
         // Check if the server script exists at given path
         if (false === $this->serverPath || false === is_file($this->serverPath)) {
-            throw new \RuntimeException(sprintf(
-                "Could not find server script at path '%s'",
-                $this->serverPath
-            ));
+            throw new \RuntimeException(
+                sprintf(
+                    "Could not find server script at path '%s'",
+                    $this->serverPath
+                )
+            );
         }
 
-        // Create process object if neccessary
+        // Create process object if necessary
         if (null === $process) {
-            $processBuilder = new ProcessBuilder(array(
-                $this->nodeBin,
-                $this->serverPath,
-            ));
-            $processBuilder->setEnv('HOST', $this->host)
-                ->setEnv('PORT', $this->port);
+            $env = array(
+                'HOST' => $this->host,
+                'PORT' => $this->port,
+            );
 
             if (!empty($this->nodeModulesPath)) {
-                $processBuilder->setEnv('NODE_PATH', $this->nodeModulesPath);
+                $env['NODE_PATH'] = $this->nodeModulesPath;
             }
 
             if (!empty($this->options)) {
-                $processBuilder->setEnv('OPTIONS', json_encode($this->options));
+                $env['OPTIONS'] = json_encode($this->options);
             }
 
-            $process = $processBuilder->getProcess();
+            $process = new Process(
+                array(
+                    $this->nodeBin,
+                    $this->serverPath,
+                ),
+                null,
+                $env
+            );
+            $process->inheritEnvironmentVariables();
```

Comment:
Causes https://travis-ci.org/minkphp/MinkZombieDriver/jobs/502013251 failure.

Reply:
I'm not sure how to solve this. PHP 5.4 doesn't have this method for Process.

Reply:
You need to use `ProcessBuilder` conditionally. Better yet check if `Process::inheritEnvironmentVariables` method exists and then:

* use your new code (this means that `ProcessBuilder` is likely deprecated)
* otherwise use old code using `ProcessBuilder`

.

Reply:
You can even keep `$env` variable and when we need to use process builder do a foreach and call `$processBuilder->setEnv`.